src/core/api/languageTools.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+import { callApi } from 'core/api';
+import type { ApiStateType } from 'core/reducers/api';
+
+
+export type LanguageToolsParams = {|
+  api: ApiStateType,
+|};
+
+export function languageTools({ api }: LanguageToolsParams) {
+  return callApi({
```

Comment:
This should set `auth: true` so that the user's token will be sent in the request in case they're signed in . I'm really not sure why `auth` defaults to `false` in `callApi()`. If you meant to never send a login token then this should explicitly set `auth: false`.

Reply:
Can we fill an issue about this if no issue exists yet ? It seems we all get caught by that. I would suggest to rename this to `sendCredentials` or something that is less misleading than `auth`.

Reply:
Yes, that would be great. I am not aware of an issue so please file it.